code/Coding_Abgabe_I/stylesheet.css

```diff
@@ -0,0 +1,181 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;500;600;900&display=swap');
+
+body {background-color:	rgb(177, 212, 195);

```

Comment:
Here and in some other parts I would recommend spliting the code after `{` to a new row:
`body {`
`background-color: rgb(177, 212, 195);`